src/chart/helper/Line.ts

```diff
@@ -261,7 +261,9 @@ class Line extends graphic.Group {
                 ? lineData.getName(idx)
                 : isFinite(rawVal)
                 ? round(rawVal)
-                : rawVal) + ''
+                : rawVal) + '',
+            // @ts-ignore
+            z: seriesModel.getData(lineData.dataType).getItemModel(idx).get('z') + 1
```

Comment:
not only `markLine` use `Line.ts`

Also `@ts-ignore` should always be avoided unless the code is very simple and you are very sure about the input type and output type.a